packages/editor/src/plugins/image/components/pixabay-image-search.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { ChangeEvent, useState, useEffect } from 'react'
+
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { cn } from '@/helper/cn'
+
+interface PixabayResponse {
+  total: number
+  totalHits: number
+  hits: PixabayImage[]
+}
+interface PixabayImage {
+  id: number
+  webformatURL: string
+  tags: string
+  previewURL: string
+  largeImageURL: string
+}
+
+interface PixabayImageSearchProps {
+  onSelectImage: (imageUrl: string) => void
+}
+
+export const PixabayImageSearch = ({
+  onSelectImage,
+}: PixabayImageSearchProps) => {
+  const [query, setQuery] = useState('')
+  const [images, setImages] = useState<PixabayImage[]>([])
+  const [isLoadingImage, setIsLoadingImage] = useState(false)
+  const [isSearching, setIsSearching] = useState(false)
+
+  const [showTags, setShowTags] = useState(true)
+
+  const editorStrings = useEditorStrings()
+  const imageStrings = editorStrings.plugins.image
+  const apiKey = 'secret'
+
+  useEffect(() => {
+    if (isLoadingImage) {
+      document.body.classList.add('overflow-hidden')
+    } else {
+      document.body.classList.remove('overflow-hidden')
+    }
+  }, [isLoadingImage])
```

Comment:
Not sure why this is needed, but maybe this should have a cleanup function to make sure on unmount the body is def. scrollable again.
``` ts
//...
return () => {
    document.body.classList.remove('overflow-hidden')
};
```

Reply:
Not needed, its a remnant, removed now, thanks !